portingdb/load_data.py

```diff
@@ -297,3 +297,13 @@ def load_from_directories(data, directories):
             package['ftbfs_age'] = max(CURRENT_FEDORA - releasever, 0)
         else:
             package['ftbfs_age'] = 0
+
+    # Consolidate non-Python requirers
+    for name, package in packages.items():
+        nonpy_requirers = {}
+        package.setdefault('needs_py2_command', False)
```

Comment:
I'Ve tried to remove this line and it seems the website works just fine without. Will check if there are any differences.

Reply:
You're right, that's a merge artifact.